File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        my_list = []
        my_dict = {} # stores value : index
        for i, num in enumerate(nums):
            if ((target - num) in my_dict):
                my_list.append(my_dict[target-num])
                my_list.append(i)
                return my_list
            else:
                my_dict[num] = i
        return my_list

File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> _sort = {0, 0 , 0};
        for (int i = 0; i < nums.size(); i++)
            _sort[nums[i]] += 1; 
        int j = 0;
        for (int i = 0; i < _sort.size(); i++)
        {
            while (_sort[i]-- > 0)
                 nums[j++] = i;
        }
    }
};